praw/models/reddit/modmail.py

```diff
@@ -8,13 +8,97 @@ class ModmailConversation(RedditBase):
 
     STR_FIELD = 'id'
 
+    @classmethod
+    def parse(cls, data, reddit):
+        """Return an instance of ModmailConversation from ``data``.
+
+        :param data: The structured data.
+        :param reddit: An instance of :class:`.Reddit`.
+
+        """
+        conversation = data['conversation']
+
+        conversation['authors'] = [reddit._objector.objectify(author)
+                                   for author in conversation['authors']]
+        conversation['owner'] = reddit._objector.objectify(
+            conversation['owner'])
+
+        if data['user']:
+            cls._convert_user_summary(data['user'], reddit)
+            conversation['user'] = reddit._objector.objectify(data['user'])
+        conversation.update(cls._convert_conversation_objects(data, reddit))
+
+        conversation = reddit._objector.snake_case_keys(conversation)
+
+        return cls(reddit, _data=conversation)
+
+    @classmethod
+    def _convert_conversation_objects(cls, data, reddit):
+        """Convert messages and mod actions to PRAW objects."""
+        result = {'messages': [], 'modActions': []}
+        # pylint: disable=redefined-builtin
+        for object in data['conversation']['objIds']:
+            key = object['key']
+            object_data = data[key][object['id']]
+            result[key].append(reddit._objector.objectify(object_data))
+        return result
+
+    @classmethod
+    def _convert_user_summary(cls, data, reddit):
```

Comment:
Also prefer a static method over class method since we don't leverage the class at all here.